sdk/appcenter/src/main/java/com/microsoft/appcenter/AppCenter.java

```diff
@@ -308,6 +309,15 @@ public static void setCountryCode(String countryCode) {
         DeviceInfoHelper.setCountryCode(countryCode);
     }
 
+    /**
+     * Set the country code or any other string to identify residency region.
+     *
+     * @param dataResidencyRegion residency region code.
+     */
+    public static void setDataResidencyRegion(@Nullable String dataResidencyRegion) {
```

Comment:
Why @Nullable here? Do we have any use case of `AppCenter.setDataResidencyRegion(null)`